draft-stark-expect-ct.md

```diff
@@ -508,12 +508,61 @@ UA SHOULD report the failure as follows:
+comply with the UA's CT policy, and add the Expect-CT header in enforcing mode
+with a long `max-age`.  Implementing user agents would note this as an Expect-CT
+Host (see  {{noting-expect-ct}}). After having done this, the configuration
+could then be reverted to not comply with the CT policy, prompting failures.
+Note this scenario would require the attacker to have substantial control over
+the infrastructure in question, being able to obtain different certificates,
+change server software, or act as a man-in-the-middle in connections.
+
+Site operators could themselves only cure this situation by one of:
+reconfiguring their web server to transmit SCTs using the TLS extension defined
+in Section 3.3 of {{!RFC6962}}, obtaining a certificate from an alternative
+certificate authority which provides SCTs by one of the other methods, or by
+waiting for the user agents' persisted notation of this as an Expect-CT host to
+reach its `max-age`. User agents may choose to implement mechanisms for users to
+cure this situation, as noted in {{usability-considerations}}.
+
 ## Maximum max-age
 
-1 year?
+There is a security trade-off in that low maximum values provide a narrow window
+of protection for users that visit the Known Expect-CT Host only infrequently,
+while high maximum values might result in a denial of service to a UA in the
+event of a hostile header attack.
+
+There is probably no ideal maximum for the `max-age` directive. Since Expect-CT
+is primarily a policy-expansion and investigation technology rather than an
+end- user protection, a value on the order of 30 days (2,592,000 seconds) may be
+considered a balance between these competing security concerns.
+
+## Avoiding amplification attacks
+
+Another kind of hostile header attack uses the `report-uri` mechanism on many
+hosts not currently exposing SCTs as a method to cause a denial-of-service to
+the host receiving the reports. If some highly-trafficked websites emitted
+a non-enforcing Expect-CT header with a `report-uri`, implementing UAs' reports
+could flood the reporting host. It is noted in {{the-report-uri-directive}} that UAs
+should limit the rate at which they emit reports, but an attacker may alter the
+Expect-CT header's fields to induce UAs to submit different reports to different
+URIs to still cause the same effect.
 
 # Privacy Considerations
 
+Expect-CT can be used to infer what Certificate Transparency policy is in use,
+by attempting to retrieve specially-configured websites which pass one user
+agents' policies but not another's. Note that this consideration is true of UAs
+which enforce CT policies without Expect-CT as well.
+
+Additionally, reports submitted to the `report-uri` could reveal information to
+a third party about which webpage is being accessed and by which IP address, by
+using individual `report-uri` values for individually-tracked pages. This
+information could be leaked even if client-side scripting were disabled.
+
```

Comment:
There's also the HPKP super-cookie consideration in https://tools.ietf.org/html/rfc7469#section-5. I think the same thing applies here, do you agree?